test/uk/gov/hmrc/plasticpackagingtax/returns/forms/ConvertedPackagingCreditSpec.scala

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.returns.forms
+
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.data.FormError
+import uk.gov.hmrc.plasticpackagingtax.returns.forms.ConvertedPackagingCredit.{
+  aboveMaxError,
+  creditEmptyError,
+  invalidFormatError,
+  maxTotalCredit,
+  totalInPence
+}
+
+class ConvertedPackagingCreditSpec extends AnyWordSpec with Matchers {
+
+  "Converted packaging credit validation rules" should {
+
+    "return success" when {
+
+      "totalInPence is withing allowed range" in {
+
+        val input = Map(totalInPence -> "100")
+
+        val form = ConvertedPackagingCredit.form().bind(input)
+        form.errors.size mustBe 0
+      }
+
+      "totalInPence allows decimal values" in {
```

Comment:
Do we need to check that we allow two decimal places only? e.g. `100.99` is allowed, `100.999` is not allowed.

Reply:
yeap agree, added test.